src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -29,7 +29,7 @@
     protected String courseId;
     protected String feedbackSessionName;
     protected FeedbackSubmissionEditPageData data;
-    protected String hasResponses;
+    protected boolean isHasResponse;
```

Comment:
The default value of a boolean class attribute is false. It shouldn't be initialised for each question, because a single response to any question should result in the user being considered as a respondent